docs/advanced-usage/user-quotas/index.md

```diff
@@ -0,0 +1,144 @@
+# User quotas
+
+Since
+[REANA 0.8.0](https://blog.reana.io/posts/2021/release-0.8.0/#cpu-and-disk-quota-accounting),
+it is possible to limit the amount of disk space and CPU time available to
+execute workflows with the help of user quotas.
+
+## Check usage and quota limits
+
+User quotas are optional features of REANA which might not be enabled in your
```

Comment:
... which may or may not be enabled in your cluster.  You can check which resources are enabled in your cluster:

```console
$ reana-client quota-show --resources
cpu
disk
```

You can check the limits imposed by the cluster administrator as follows:

```console
$ reana-client quota-show --resource disk --report limit -h
No limit.
$ reana-client quota-show --resource cpu --report limit -h
1h
```

We see that this particular REANA example instance does not impose any limits on the disk usage; however the total runtime for all your workflows is limited to 1 hour only.

(Comment: this example is a bit unusual, because there would usually be some disk limit, and moreover the examples below are showing how to delete big files, which implies there is some disk usage quota.  (So the page narrative is a bit broken.) Also, the quota of 1h is too small for physics usage, so people may be confused whether this is for one run or for all runs. We could perhaps bump the CPU quota numbers, but hopefully people will understand that this is just an example, since I agree that it would be good to show in the docs also "unlimited" option! Hmm, perhaps we could show the disk quota limit to keep the page narrative, and show in a new section that cluster admins may also select "unlimited" so that you wolud see "No limit" etc. WDYT? We can muse IRL.)

You can inspect your current usage of the resources as follows:

```console
$ reana-client quota-show --resource disk -h
192 KiB
$ reana-client quota-show --resource cpu -h
1m 19s out of 1h used (2%)
```

(Comment: it looks a bit weird to show values without units, so I think we can focus on `-h` directly, or else always show both examples. The current example "192512 used" is very ambiguous)

(and now we can transition to the Web UI section...)

Reply:
> (Comment: this example is a bit unusual, because there would usually be some disk limit, and moreover the examples below are showing how to delete big files, which implies there is some disk usage quota. (So the page narrative is a bit broken.)

I have added a limit of 500 GiB on the disk usaget and a "Note" box to show the output of reana-client when there are no limits.

> Also, the quota of 1h is too small for physics usage, so people may be confused whether this is for one run or for all runs.

The CPU time limit is now 10000 hours.